src/main/java/com/questhelper/quests/gertrudescat/GertrudesCat.java

```diff
@@ -0,0 +1,219 @@
+
+		steps.put(0, talkToGertrude = getTalkToGertrude());
+		steps.put(1, talkToChildren = getTalkToChildren());
+		steps.put(2, getLumberyard());
+		steps.put(3, getFeedCat());
+		steps.put(4, findFluffsKitten());
+		steps.put(5, finishQuest = returnToGertrude());
+		return steps;
+	}
+
+	private NpcStep returnToGertrude()
+	{
+		return new NpcStep(this, NpcID.GERTRUDE,
+			new WorldPoint(3148, 3413, 0), "Talk to Gertrude.");
+	}
+
+	private QuestStep findFluffsKitten()
+	{
+
+		//Need to find to ways to hide arrow
+		searchNearbyCrates = new NpcStep(this, NpcID.CRATE,
+			"Search for a kitten.", true);
+		ObjectStep climbDownLadderStep = goDownLadderStep();
+		ObjectStep climbUpLadderStep = getClimbLadder();
+		ArrayList<ItemRequirement> fluffsKittenRequirement = new ArrayList();
+		fluffsKittenRequirement.add(new ItemRequirement("Fluffs Kitten", ItemID.FLUFFS_KITTEN));
+
+		climbUpLadderStep.addItemRequirements(fluffsKittenRequirement);
+		Conditions hasFluffsKittenUpstairs = new Conditions(hasFluffsKitten, isUpstairsLumberyard);
+
+		giveKittenToFluffy = getGertrudesCat("Return the kitten to Gertrude's Cat", new ItemRequirement("Fluffs Kitten", ItemID.FLUFFS_KITTEN));
+		giveKittenToFluffy.addIcon(ItemID.FLUFFS_KITTEN);
+
+		ConditionalStep conditionalKitten = new ConditionalStep(this, searchNearbyCrates);
+		conditionalKitten.addStep(hasFluffsKittenUpstairs, giveKittenToFluffy);
+		conditionalKitten.addStep(hasFluffsKitten, climbUpLadderStep);
+		conditionalKitten.addStep(isUpstairsLumberyard, climbDownLadderStep);
+
+		searchNearbyCrates.addSubSteps(climbDownLadderStep);
+		giveKittenToFluffy.addSubSteps(climbUpLadderStep);
+
+		return conditionalKitten;
+	}
+
+	private ObjectStep goDownLadderStep()
+	{
+		return new ObjectStep(this, ObjectID.LADDER_11795, new WorldPoint(3310, 3509, 1)
+			, "Climb down ladder");
+	}
+
+	private QuestStep getFeedCat()
+	{
+		gertrudesCat2 = getGertrudesCat("Use seasoned Sardine on Gertrudes cat.", null);
+		gertrudesCat2.addIcon(ItemID.SEASONED_SARDINE);
+
+		ObjectStep climbLadder = new ObjectStep(this, ObjectID.LADDER_11794,
+			new WorldPoint(3310, 3509, 0), "Climb ladder");
+
+		ConditionalStep lumberyard = new ConditionalStep(this, climbLadder, seasonedSardine);
+		lumberyard.addStep(isUpstairsLumberyard, gertrudesCat2);
+		gertrudesCat2.addSubSteps(climbLadder);
+
+		return lumberyard;
+	}
+
+	private QuestStep getLumberyard()
+	{
+		gertrudesCat = getGertrudesCat("Use bucket of milk on Gertrudes cat.", new ItemRequirement("Bucket of milk", ItemID.BUCKET_OF_MILK));
+		gertrudesCat.addIcon(ItemID.BUCKET_OF_MILK);
+
+		ObjectStep climbLadder = getClimbLadder();
+
+		ConditionalStep lumberyard = new ConditionalStep(this, climbLadder, seasonedSardine);
+		lumberyard.addStep(isUpstairsLumberyard, gertrudesCat);
+		gertrudesCat.addSubSteps(climbLadder);
+
+		return lumberyard;
+	}
+
+
+	private NpcStep getGertrudesCat(String text, ItemRequirement requirement)
```

Comment:
If you're wanting to structure the quest like this, it'd be a bit more succinct if you instead just made this function:

```
private NpcStep getGertrudesCat(String text, ItemRequirement... requirement)
{
    return new NpcStep(this, NpcID.GERTRUDES_CAT_3497,
				new WorldPoint(3308, 3511, 1), text, requirement);
}
```